setup.py

```diff
@@ -130,7 +133,7 @@ def main():
         install_requires=ldict['REQUIRES'],
         setup_requires=['future', 'configparser'],
         provides=ldict['PROVIDES'],
-        packages=find_packages(exclude=['*.tests']),
+        packages=find_packages(),
```

Comment:
@oesteban, @chrisfilo - any idea why the tests directories were removed from setup? i'm putting them back in.